webdev5.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Leaflet Map with a Feature Layer</title>
+    <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
+
+    <!-- Load Leaflet from CDN-->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.4/dist/leaflet.css" />
+
+    <script src="https://unpkg.com/leaflet@1.3.4/dist/leaflet.js"></script>
+
+    <!-- Load Esri Leaflet from CDN -->
+    <script src="https://unpkg.com/esri-leaflet@2.2.3/dist/esri-leaflet.js"></script>
+
+    <!-- Load Esri Leaflet Renderers from CDN -->
+    <script src="https://unpkg.com/esri-leaflet-renderers@2.0.6/dist/esri-leaflet-renderers.js"></script>
+
+    <style>
+        html,
+        body,
+        #map {
+            height: 100%;
+            width: 100%;
+            margin: 0;
+            padding: 0;
+        }
+
+    </style>
+</head>
+
+<body>
+    <header>
+        <h1>Minnesota County Boundaries</h1>
+        <h3>GIS 4533 Web-based GIS <br> Keely Hansen <br> Source: MN Geospatial Commons</h3>
+    </header>
+
+    <div id="map"></div>
+
+    <script>
+        // add map
+        const map = L.map('map', {
+            center: [46.471032, -94.391392],
+            zoom: 7
+        });
+
+        // Add new base map layer to map from http://leaflet-extras.github.io/leaflet-providers/preview/
+        L.tileLayer('https://server.arcgisonline.com/ArcGIS/rest/services/Canvas/World_Light_Gray_Base/MapServer/tile/{z}/{y}/{x}', {
+            attribution: 'Tiles &copy; Esri &mdash; Esri, DeLorme, NAVTEQ'
+        }).addTo(map);
+
+        // add hosted feature layer
+        const MNCountyBoundariesLayer = L.esri.featureLayer({
+            url: 'https://services.arcgis.com/8df8p0NlLFEShl0r/arcgis/rest/services/Map_Boundaries/FeatureServer/0'
+        }).addTo(map);
+
+        // set up popup for MN County Boundaries layer
+        MNCountyBoundariesLayer.bindPopup(function(evt) {
+            return L.Util.template(
+                '<h3>{CTY_NAME}</h3>', evt.feature.properties);
```

Comment:
Damn dude! 😮 Advanced work here making use of a Leaflet utility we haven't worked with before!
Great job! 👍

Reply:
Oh, I guess this was what I had in the example. Still, you had to figure out how to update it. 😸